docs/commands.md

```diff
@@ -124,6 +125,34 @@ Try to fetch transformed JS code from the global cache, if configured.
 
 Path to the CLI configuration file.
 
+### `clean`
+
+Usage:
+
+```sh
+react-native clean
+```
+
+Clean caches. Commonly used to ensure build failures are not due to stale cache.
```

Comment:
```suggestion
Cleans caches. Commonly used to ensure build failures are not due to stale cache. By default, it will prompt which caches to purge, with Watchman and Metro already checked.
```